apps/interactive-calibration/parametersController.cpp

```diff
@@ -134,7 +128,7 @@ bool calib::parametersController::loadFromParser(cv::CommandLineParser &parser)
     }
     else if(templateType.find("charuco", 0) == 0) {
         mCapParams.board = chAruco;
-        mCapParams.boardSize = cv::Size(6, 8);
+        mCapParams.boardSize = cv::Size(5, 7);
```

Comment:
no need to touch it.

Reply:
After fix the vars `h`/`w` is setting num corners for CharucoBard (like in chessboard)
After fix CharucoBoard create looks like this:
cv::aruco::CharucoBoard::create(mBoardSize.width **+ 1**, mBoardSize.height **+ 1**, capParams.charucoSquareLength,
                                                        capParams.charucoMarkerSize, mArucoDictionary);

`mCapParams.boardSize = cv::Size(5, 7);` was fixed to support compatibility with old default parameters